data/binding/disableableBinding.go

```diff
@@ -0,0 +1,61 @@
+package binding
+
+import (
+	"log"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/data/binding"
+)
+
+type disableableBinding struct {
+	binding.Bool
+
+	inverted bool
+	targets  []fyne.Disableable
+}
+
+// NewDisableableBinding returns a `Bool` binding which accepts Disableable targets.
+// When the Bool changes, the targets Enable or Disable method will be executed.
+func NewDisableableBinding(targets ...fyne.Disableable) *disableableBinding {
+	newBinding := &disableableBinding{
+		Bool:    binding.NewBool(),
+		targets: targets,
+	}
+
+	// Add default listener
+	newBinding.AddListener(binding.NewDataListener(newBinding.update))
+
+	return newBinding
+}
+
+// Adding targets to the binding.
+// This will update the Disable status of the targets immediately.
+func (b *disableableBinding) AddTargets(targets ...fyne.Disableable) {
+	b.targets = append(b.targets, targets...)
+	b.update()
+}
+
+// Invert will switch the behavior of when the targets will be Enabled or Disabled.
+// This will update the Disable status of the targets immediately.
+func (b *disableableBinding) Invert(inverted bool) {
```

Comment:
Shouldn't this be `SetInverted`? The name `Invert` applies action, which is to take something and make it the opposite state - which isn't really what the method does...

Reply:
Good idea with SetInverted. I had it as an action first, but then realized that it wasn't a good solution so I changed it. I'll change the name.
Isn't Widgets to generalized since it's only widgets that implements Disableable that are accepted? I agree that "targets" isn't a good name either. I guess "disableableWidgets" too long?

Not really sure what similar scenarios there are in Fyne and how they are named.

Reply:
In Go we don't typically put the type in the naming do we? For example `Form.AddItem` takes a `widget.FormItem` etc.
So the type ensures safety and the "Disablable" context is kindof provided by the object you are working with.
Happy to get others thoughts of course.

Reply:
I've changed the name to Widget and also a SetInverted.
Do you feel that this issue is resolved or was there something more about name?
I've been thinking about maybe changing the name of the binding. Fells a bit to much to say binding.DisableableBinding.
Either change the name to Disableable or DisableableBool.
Any opinions on this?

Reply:
I agree with you it could have a smoother name. `binding.NewDisableable` is a pretty good suggestion.

Apologies for the delay